Markdown/Markdown.sublime-syntax

```diff
@@ -855,6 +855,58 @@ contexts:
     - match: |-
          (?x)
           {{fenced_code_block_start}}
+          ((?i:xml))
+          \s*$          # any whitespace until eol
+      captures:
+        1: markup.raw.code-fence.xml.markdown punctuation.definition.raw.code-fence.begin.markdown
+        2: markup.raw.code-fence.xml.markdown constant.other.language-name.markdown
+      embed: scope:text.xml
+      embed_scope: markup.raw.code-fence.xml.markdown
+      escape: '^[ ]{,3}\1\s*$'
+      escape_captures:
+        0: markup.raw.code-fence.xml.markdown punctuation.definition.raw.code-fence.end.markdown
```

Comment:
Only the fences should get the punctuation scope, not any whitespace surrounding it.